os/arch/arm/src/amebad/amebad_serial.c

```diff
@@ -564,6 +564,13 @@ static int rtl8721d_up_ioctl(FAR struct uart_dev_s *dev, int cmd, unsigned long
 		rtl8721d_up_rxint(dev, priv->rxint_enable);
 		break;
 
+	case TIOCLOOPBACK:
+		if (!termiosp) {
```

Comment:
All cases check that `termiosp` is NULL or not.
How about making it as common at line 510?

Reply:
i think, we should not do that because in future, suppose if someone wants to send 0 in termios and just wants to work on cmd then it will return error.
so i think we should not change this.